adapter/internal/oasparser/model/policy_layout.go

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package model
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/wso2/product-microgateway/adapter/internal/oasparser/constants"
+)
+
+// supportedPoliciesMap maps (policy action name) -> (policy layout)
+var supportedPoliciesMap = map[string]policyLayout{
+	"SET_HEADER": {
```

Comment:
Constants here?

Reply:
These are only used here in the Adapter. This action name is used only in Enforcer.